src/runtime/storage/store-info.ts

```diff
@@ -153,6 +154,46 @@ export class StoreInfo<T extends Type> implements Comparable<StoreInfo<T>> {
     });
     return builder.toString();
   }
+
+  static isSingletonInterfaceStore(store: StoreInfo<Type>): store is StoreInfo<SingletonInterfaceType> {
```

Comment:
Why do we need the static versions of the member methods?

Reply:
probably this is my ignorance, but 
`stores.find(StoreInfo.isSingletonInterfaceStore)` returns an object of type `StoreInfo<SingletonInterfaceType>` while
`stores.find(s => s.isSingletonInterfaceStore())` returns an object of type StoreInfo<Type>, which then needs to be casted.
any ideas how to avoid the static method? I'll be happy to send a follow up PR.

Reply:
Not sure how else to get around that. Would be helpful to have a comment explaining this for future readers.